gpc-api-facade/src/main/resources/logback.xml

```diff
@@ -0,0 +1,15 @@
+<configuration>
+    <appender name="TEXT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>${PS_LOGGING_FORMAT:-%d{yyyy-MM-dd HH:mm:ss.SSS} Level=%-5level Logger=%logger{36} ConversationId=%X{ConversationId} Thread="%thread" Message="%msg"%n}
+            </pattern>
+        </encoder>
+    </appender>
+
+
+    <root level="${PS_LOGGING_LEVEL:-INFO}">
+        <appender-ref ref="TEXT}"/>
```

Comment:
Starting to think this might in intentional? 😄  just flagging incase

Reply:
good catch - fixed ;)